### DISCUSSION TITLE
Laravel voyager and Git&Github ?

### DISCUSSION THREAD

**Discussion by abdelghani1002** [thedevdojo/voyager#5842] [Q&A] [answered]
#5749
I have a problem with laravel voyager project, When I add some new changes like new BREADs to my project, they don't be readed by git , the logic of the new BREADs is keeped in vendor/tcg/voyager/somewhere. 
I need the new changes to be readed by git, HOW to make them ???
Thank you.
hope I explain my problem well. ( I'm stell beginner in English ).

**Comment by Emerica** [comment]
BREAD is tracked in database.  
Only files are used if you manaully override BREAD in which case git would find them unless you somehow added the directory to .gitignore.

**Reply by abdelghani1002** [reply]
thank you for your reply.
Is there some way to make git read just the new BREADs ?

**Reply by Emerica** [reply, marked as answer]
If you use something like a Iseed generation package you could export migrations and seeds and add them to git that way.
